avocado/plugins/runners/podman_image.py

```diff
@@ -0,0 +1,71 @@
+import asyncio
+import time
+from multiprocessing import Process, SimpleQueue
+
+from avocado.core.nrunner.app import BaseRunnerApp
+from avocado.core.nrunner.runner import RUNNER_RUN_STATUS_INTERVAL, BaseRunner
+from avocado.core.utils import messages
+from avocado.utils.podman import Podman, PodmanException
+
+
+class PodmanImageRunner(BaseRunner):
+    """Runner for dependencies of type podman-image
+
+    This runner handles download and verification.
+
+    Runnable attributes usage:
+
+     * kind: 'podman-image'
+
+     * uri: the name of the image
+
+     * args: not used
+    """
+
+    name = 'podman-image'
+    description = f'Runner for dependencies of type {name}'
+
+    def _run_podman_pull(self, uri, queue):
+        try:
+            podman = Podman()
+            loop = asyncio.get_event_loop()
+            loop.run_until_complete(podman.execute('pull', uri))
+            result = 'pass'
+        except PodmanException:
+            result = 'fail'
+        queue.put({'result': result})
+
+    def run(self, runnable):
+        # pylint: disable=W0201
+        self.runnable = runnable
+        yield messages.StartedMessage.get()
+
+        if not runnable.uri:
+            reason = 'uri identifying the podman image is required'
+            yield messages.FinishedMessage.get('error', reason)
+        else:
+            queue = SimpleQueue()
+            process = Process(target=self._run_podman_pull,
+                              args=(runnable.uri, queue))
+            process.start()
+            while queue.empty():
+                time.sleep(RUNNER_RUN_STATUS_INTERVAL)
+                yield messages.RunningMessage.get()
```

Comment:
`Podman` utility uses its own logs, which will be lost here. IMO, we should use `messages.start_logging` to catch logs and send them to avocado.

Reply:
I solved this in a different way, by silencing the "avocado.utils.podman" logger in this specific process.  The relevant error messages are already caught and shown by the exception handler.